src/components/shared/formItems/ProjectHandle.tsx

```diff
@@ -71,13 +73,17 @@ export default function ProjectHandle({
 
   const checkHandle = useCallback(
     (rule: any, value: any) => {
+      // Input field is back at it's initial state
+      if (initialValue && initialValue === value)
+        return Promise.resolve()
```

Comment:
@aeolianeth just looked at your PR - you might need to add this little check, but your solution is in line  with and way more holistic than this one so I'll close this PR in favor of yours.